stackslib/src/net/api/callreadonly.rs

```diff
@@ -194,6 +194,7 @@ impl RPCRequestHandler for RPCCallReadOnlyRequestHandler {
         preamble: HttpRequestPreamble,
         contents: HttpRequestContents,
         node: &mut StacksNodeState,
+        _ibd: bool,
```

Comment:
You don't have to modify all of the calls to `try_handle_request()`.  You could instead add it as a public field to `StacksNodeState`, so interested callers can access it.

This is the purpose of the `StacksNodeState` struct -- to pass state about the node to RPC handlers in a forwards-compatible manner.